### ISSUE TITLE
looting bot

### ISSUE THREAD

**Issue by Bitcorn69** [coccoinomane/crabada.py#3]
can we do a discord chat about the looting bot?

**Comment by coccoinomane** [comment]
Hey @Bitcorn69 , sorry for the late reply.
As a consequence of the anti-bot patch released last week by Crabada, I am evaluating whether and how to proceed with the looting bot.
I'll update the readme when I have a plan :-)
PS: The mining bot on Github (this repository) is still working though and can be used for free.
Cheers,
Guido

**Comment by mzahirr** [comment]
Hi @coccoinomane is there any progress for looting bot?

**Comment by nikojaro** [comment]
> 

I don't think finding a loot will be fixed. Reinforce and settle can be fixed.

**Comment by Bitcorn69** [comment]
What about solving the captchas manually? There are also services to solve captchas with api but they are not fast enough...they have humans solving them and it costs per captcha.

**Comment by mzahirr** [comment]
> What about solving the captchas manually? There are also services to solve captchas with api but they are not fast enough...they have humans solving them and it costs per captcha.

I agree . Is there any way ?

**Comment by coccoinomane** [comment]
Hey guys, thank you for your inputs!

The more I think about the looting bot, the more I lean towards NOT going forward with it.

There are very good technical reasons that make the task daunting: reliance on third-party APIs to bypass the captcha makes it a non trivial implementation, and playing catch-up with Crabada's devs as they improve the captcha is very time consuming.

But, more than that, I love the Crabada game and I respect the Crabada community.

Before the captcha was implemented, making a looting bot was fair game: it was nothing else than interacting with the blockchain. Everything on the blockchain is permissionless, so I did not have any qualms to make a looting bot, and I even advertised it in the README.

Now, however, the team has gone a long way towards implementing anti-bot features, for a reason: bots have started to scare humans away, which is not good for Crabada and its community. After all, we all rely on Crabada, botters and normal players alike.

It is one thing to play by the blockchain rules, it is a whole another thing to crack a captcha. By bypassing the captcha I feel like betraying the trust of Crabada devs, and, to be honest, I would not blame them if they started banning players and crabs.

Therefore, as @nikohd12 suggested, I think I will gradually remove the `attack` feature (which does not work properly anyway 😅) and, as far as looting is concerned, keep only the `reinforceAttack` and `closeLoots` features.
I think that these two features can still help looters automate away a big part of their tasks, which is a good thing that does not hurt the community.

Obviously, I will also keep all mining-related functions.

Cheers,
Cocco
